### ISSUE TITLE
[a11y]: two InterstitialScreen component Accessibility Checker errors (probably one-liner)

### ISSUE THREAD

**Issue by 2312323123** [carbon-design-system/ibm-products#7263]
### Package

@carbon/ibm-products

### Browser

Chrome

### Operating System

MacOS

### Package version

v2.63.0

### React version

v17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker, and for the first issue also the one built-in into storybook

### Description

There's an error when running IBM Equal Accessibility Checker on site with the current version of the InterstitialScreen component, regarding element with class
c4p--carousel__elements
The violation is "The scrollable element <div> with non-interactive content is not tabbable"
One way to remove this violation is by giving it tabindex="0".

And there's another error in element with class
c4p--interstitial-screen--body
The violation is "The tabbable element does not have a valid widget role"
which can be mitigated by removing its tabindex="0"

The first (c4p--carousel__elements) one is visible in https://ibm-products.carbondesignsystem.com/?path=/story/experimental-onboarding-interstitial-screen-interstitialscreen--interstitial-screen-full-screen-no-media-with-cards&globals=viewport:basic
You can see it in the Accessibility tab as "Scrollable region must have keyboard access" 

So the issue seems to be solvable by moving that tabindex="0" a few levels deeper from c4p--interstitial-screen--body to c4p--carousel__elements.

### WCAG 2.1 Violation

Success Criterion 4.1.2: Name, Role, Value

### Reproduction/example

https://stackblitz.com/edit/github-m8ct9l9j-mii4sxzt?file=src%2FExample%2Fimage.js

### Steps to reproduce

it appears in our case when IntersititalScreen is visible and IBM Equal Access Accessibility Checker is run. In the stackblitz IBM Equal Access Accessibility Checker probably doesn't check contents of the iframe, sadly.

Here's a screen of what I get (these are the last two violations I'm tasked to investigate/fix):

![Image](https://github.com/user-attachments/assets/610ada41-22ab-47ea-8b58-cec223ca3d69)

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/ibm-products/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/ibm-products/issues) for duplicate problems

**Comment by amal-k-joy** [comment]
Closing as not planned.
